src/mapDataManipulation/runEffectFunctions.ts

```diff
@@ -29,6 +33,9 @@ export async function runEffectFunctions(originalBlob: Blob, effects: Effects):
   // change the values according to the algorithm (gamma)
   rgbMappingArrays = runGammaEffectFunction(rgbMappingArrays, effects);
 
+  // change the values according to the algorithm (color manipulation)
+  rgbMappingArrays = runSimpleColorEffectFunction(rgbMappingArrays, effects);
```

Comment:
I think the name `runColorEffectFunction` would be more understandable? Fftr.

Reply:
Thanks, I guess so, I don't have a strong opinion because I don't know how the more advanced one will be named or how exactly it will be implemented.

Changed it to `runColorEffectFunction` as suggested.